README.md

```diff
@@ -6,10 +6,11 @@
 [![Version][version-badge]][version-url]
 [![API Reference][api-badge]][api-url]
 [![Go version][go-badge]][go-url]
-[![Discord chat][discord-badge]][discord-url]
 [![License][license-badge]][license-url]
 [![Sourcegraph][sg-badge]][sg-url]
 
+[![Discord chat][discord-badge]][discord-url]
```

Comment:
I assume you are creating a new paragraph here on purpose?

Reply:
yes, i wanted to separate the technical badges (golang version, testing, etc) from the social (discord) badge